content/en/docs/installation-new/_index.md

```diff
@@ -1,66 +1,299 @@
 ---
-title: Installation
-description: How to and where install OpenServerless
-weight: 50
+title: Installation-new
+description: How to install OpenServerless
+weight: 80
 ---
+
+Slogan: OpenServerless is ...
+
+The audience of this document is composed by developers, SREs, and OpenServerless contributors.
+
+
+## Prepare the target environment on which you want to install OpenServerless
+
+TODO 1: PUT A SCHEMA HERE
+TODO 2: FOR EACH EXTERNAL LINK, ADD A TARGET _blank TO OPEN THE PAGE IN A NEW BROWSER TAB.
+
+
+Here we illustrate how to prepare a target environment for OpenServerless. 
+
+Depending on your use case, you can install OpenServerless:
+
+- for *development* on a [single node](#single-node-development-installation) environment,
+    either on your local machine or on a Linux server.
+
+- for *production*, on a [multi node](#multi-node-production-installation) environment
+    provided by a Kubernetes cluster.
+
+### Single Node development installation
+
+For *development* purposes, you can install a single node
+OpenServerless deployment on:
+
+- your [local machine](/docs/installation-new/install/local/), you need
+    [Docker Desktop](/docs/installation-new/prereq/docker/)
+
+- on a *single Linux server node*
+  [Linux server](/docs/installation-new/install/server/), you need a server with [passwordless ssh access and sudo](/docs/installation-new/prereq/server/).
+
+The OpenServerless `ops` CLI can automatically install a Kubernetes environment, using
+[K3S](https://k3s.io), but if you prefer, you can install a single-node
+Kubernetes instance by yourself.
+
+If you choose to install Kubernetes on your server, we provide support
+for:
+
+- [SuSE K3S](/docs/installation-new/prereq/k3s/)
+
+- [Canonical MicroK8S](/docs/installation-new/prereq/mk8s)
+
+### Multi Node production installation
+
+For *production* purposes, you need a multi-node Kubernetes cluster
+that satisfies [those prerequisites](/docs/installation-new/prereq/kubernetes/),
+ with its `kubeconfig` file accessible.
+
+If you have such a cluster, you can install
+[OpenServerless in a Kubernetes cluster](/docs/installation-new/install/cluster/)
+
+If you do not have a cluster and you need to setup one, we provide
```

Comment:
who should provide this kind of support? Here the "we" cannot refer to the open source project..